src/services/Crypto/Mintlayer/Mintlayer.js

```diff
@@ -0,0 +1,56 @@
+import init, {
+  make_private_key,
+  public_key_from_private_key,
+  sign_message,
+  verify_signature,
+  make_default_account_pubkey,
+  make_receiving_address,
+  pubkey_to_string,
+} from './@mintlayerlib-js/wasm_crypto.js'
+
+const NETWORKS = {
+  mainnet: 0,
+  testnet: 1,
+  regtest: 2,
+  signet: 3,
+}
+
+export const getPrivateKey = async () => {
```

Comment:
can we add tests for each of these to make sure they do what we're expecting. add a mix of chosen values and "random" tests.

By which I mean we test given a constant key k and a message m we always get a signature that can be verified using the verify function. 

then some tests that do it with random values too. essentially a very basic fuzz to catch issues with random values and a regression test to make sure we don't break anything in the future